src/benchmarks/micro/libraries/System.Drawing/Perf_Image_Load.cs

```diff
@@ -66,6 +66,11 @@ private static ImageTestData[] CreateTestCases()
 
                 throw;
             }
+            catch (Exception) when (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
Would it be better to just do a check at the beginning of the test and return the first element if in OSPlatform.OSX?

Reply:
What exactly do you suggest? Replicate detection from runtime? This is essentially constructor for the tests so will publish no tests if this fails without need to understand runtime dependencies here. Unlike XUnit, there is no easy way how to skip method based on runtime detection.

Reply:
I was suggesting early detection in this constructor: https://github.com/dotnet/performance/pull/1173/files/d47ddc061dea94c6f801e59cdd2f9893c45ea4a0#diff-18cd5ab72627c912954386cf254f95a6R51

```cs
if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
    return new ImageTestData[0];

try
{
....
}
```

Reply:
With that, we could _never_ run the test on macOS. The proposed change allows running the test if you get libgdi via brew/mono-libgdiplus.

Reply:
Ah that's true. Ok then disregard 😄